language-support/scala/bindings/src/main/scala/com/digitalasset/ledger/client/binding/binding/Primitive.scala

```diff
@@ -184,29 +160,27 @@ private[client] object OnlyPrimitive extends Primitive {
   object TemplateId extends TemplateIdApi {
     // the ledger api still uses names with only dots in them, while QualifiedName.toString
     // separates the module and the name in the module with colon.
-    override def apply[Tpl <: Template[Tpl]](
+    def apply[Tpl <: Template[Tpl]](
```

Comment:
It provides the only static guarantee that this method is visible.

```suggestion
    override def apply[Tpl <: Template[Tpl]](
```

Reply:
fixed